hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryBackupOperation.java

```diff
@@ -74,7 +75,7 @@ protected void runWithPartitionScanForNative() {
             }
         }, true);
 
-        if (outComes != null) {
+        if (!isEmpty(outComes)) {
```

Comment:
Since `outcomes` is non-null already, isn't it better to remove `if` condition completely and use `while(!outComes.isEmpty()) {..}` loop instead of `do {..} while (!outComes.isEmpty());`?

Reply:
thx, fixed for both primary and backup operations.